targets/unity-v2/make.js

```diff
@@ -378,24 +417,34 @@ function getAuthParams(apiCall) {
     return "AuthType.None";
 }
 
-function getRequestActions(tabbing, apiCall) {
-    if (apiCall.name === "GetEntityToken")
+function getRequestActions(tabbing, apiCall, isApiInstance = false) {
+    if (apiCall.name === "GetEntityToken" && isApiInstance === false)
         return tabbing + "AuthType authType = AuthType.None;\n" +
             "#if !DISABLE_PLAYFABCLIENT_API\n" +
             tabbing + "if (authType == AuthType.None && PlayFabClientAPI.IsClientLoggedIn())\n" +
```

Comment:
>PlayFabClientAPI. [](start = 56, length = 17)

Please remove "PlayFabClientAPI." prefix here. This code will be used to run any other API template, so we want to keep just the static method name here.

Reply:
For example PlayFabAuthenticationAPI has no IsClientLoggedInMethod what should we do? Replace PlayFabClientAPI with PlayFabHttp?

Reply:
Please try to do what you did with developer secret key below: 

!string.IsNullOrEmpty(request.AuthenticationContext.ClientSessionKey ?? PlayFabSettings.ClientSessionKey)



---
In reply to: [262695893](https://github.com/PlayFab/SDKGenerator/pull/420#discussion_r262695893) [](ancestors = 262695893)